scripts/solve_network.py

```diff
@@ -516,7 +517,7 @@ def solve_network(n, config, opts="", **kwargs):
     max_iterations = cf_solving.get("max_iterations", 6)
 
     # add to network for extra_functionality
-    n.config = config
+    n.config = update_config(config)
```

Comment:
I'm wondering that we could add the "add_commit function" directly into the beginning of the Snakefile so that the "commit" code is automatically included in all configs that are passed to the whole workflow and hence automatically saved in every function.

What do you think?

Reply:
I think it could be usefull for debugging.
Would you add another rule for copying the commit or would you update it in the beginning of the snakefile?